docs/copilot/language-models.md

```diff
@@ -85,6 +101,21 @@ To update the provider details, such as the API key or endpoint URL:
 
 1. Update the provider details, such as the API key or endpoint URL.
 
+### Considerations
+
+There are a number of considerations when using your own language model API key in VS Code:
+
+- Embeddings are still sent to Copilot API.
+- Repository indexing is going through the Copilot API.
+- Query refinement, intent detection, and side queries are still going through the Copilot API.
+- When using your own model, there is no guarantee that responsible AI filtering is applied to the model's output.
```

Comment:
I would add 

•	Capabilities may differ from built-in models, such as tool-calling (important in agent mode!) or vision